test/integration/transactions/accountDelete.ts

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash'
+
+import { AccountDelete } from 'xrpl-local/models/transactions'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import { generateFundedWallet, ledgerAccept, testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('AccountDelete', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('base', async function () {
+    const wallet = await generateFundedWallet(this.client)
```

Comment:
nit: Using a name like `wallet2` might make this easier to differentiate from `this.wallet`